### PR TITLE
Fix mg login and refactor.

### PR DESCRIPTION
**Pull request by SuKSW**
## Purpose
- Fix https://github.com/wso2/product-apim-tooling/issues/641
- Refactor code
- Update docs

### CODE REVIEW

File: import-export-cli/cmd/mg/add.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ const (
 	addCmdLongDesc  = `Add new environment and its related endpoints to the config file`
 )
 const addCmdExamples = utils.ProjectName + " " + mgCmdLiteral + " " + addCmdLiteral + " " + envCmdLiteral +
-	" prod --host  https://localhost:9443" +
+	" prod --host https://localhost:9843" +
```
Comment (npamudika) [inline import-export-cli/cmd/mg/add.go:31]: are we keeping the flag as host? Shouldn't it be good to have it as adapter?

Comment (SuKSW) [reply import-export-cli/cmd/mg/add.go:31]: We can change the flag to --adapter. Updating this, and the docs.

Comment (npamudika) [reply import-export-cli/cmd/mg/add.go:31]: great. Let's update here too

Comment (SuKSW) [reply import-export-cli/cmd/mg/add.go:31]: Updating in add.go

**Review by npamudika — APPROVED** [review]

**Review by 1akshitha — APPROVED** [review]

**Pull request merged**